vizro-ai/tests/unit/vizro-ai/dashboard/_response_models/test_controls.py

```diff
@@ -0,0 +1,49 @@
+import pytest
+from vizro_ai.dashboard._response_models.controls import ControlPlan, _create_filter_proxy
```

Comment:
```suggestion
from vizro_ai.dashboard._response_models.controls import ControlPlan, _create_filter_proxy, _create_filter
```

Reply:
We should be able to test "_create_filter" with fakellm. You are right that model manager is needed in this case though.